package.json

```diff
@@ -21,7 +21,7 @@
         "ffmpeg-static": "^5.2.0",
         "install-peerdeps": "^3.0.3",
         "libsodium-wrappers": "^0.7.13",
-        "play-dl": "^1.9.7"
+        "play-dl": "shinray/play-dl#dist"
```

Comment:
temporary workaround (which means this will likely be here forever...)
should investigate maybe switching to a different backend, like ytdl-core?

Reply:
https://github.com/shinray/play-dl/commit/2bfbfe6decd68261747ba55800319f9906f12b03